packages/protocol/contracts/tokenvault/BridgedERC20Base.sol

```diff
@@ -33,25 +33,26 @@ abstract contract BridgedERC20Base is EssentialContract, IBridgedERC20 {
     error BB_MINT_DISALLOWED();
 
     function changeMigrationStatus(
-        address addr,
```

Comment:
Just renames below for clarity, and moved out the shared conditional to `_isMigratingOut()`.